ansible-ipi-install/inventory/hosts.sample

```diff
@@ -86,3 +86,37 @@ worker-1 name=worker-1 role=worker ipmi_user=admin ipmi_password=password ipmi_a
 # Provision Host
 [provisioner]
 provisioner.example.com
+
+# Registry Host
+#   Define a host here to create or use a local copy of the installation registry
+#   Used for disconnected installation
+[registry_host]
+# disconnected.example.com
+
+# [registry_host:vars]
+# The following cert_* variables are needed to create the certificates
+#   when creating a disconnected registry. They are not needed to use
+#   an existing disconnected registry.
+# cert_country=MyCountry
+# cert_state=MyState
+# cert_locality=MyCity
+# cert_organization=MyCompany
+# cert_organizational_unit=MyDepartment
+
+# The port exposed on the disconnected registry host can be changed from
+# the default 5000 to something else by changing the following variable.
+# registry_port=5000
+
+# The directory the mirrored registry files are written to can be modified from teh default /opt/registry by changing the following variable.
+# registry_dir="/opt/registry"
+
+# The following two variables must be set to use an existing disconnected registry.
+#
+# Specify a file that contains extra auth tokens to include in the
+#   pull-secret if they are not already there.
+# disconnected_registry_auths_file=/home/kni/mirror_auth.json
+
+# Specify a file that contains the addition trust bundle and image
+#   content sources for the local registry. The contents of this file
+#   will be appended to the install-config.yml file.
+# disconnected_registry_mirrors_file=/home/kni/ic-appends.yml
```

Comment:
These pieces should go under the cached_enabled=True variable in the inventory/hosts.sample

Reply:
i grouped these together with the registry_host section since they all have to do with the disconnected registry and not the cached_enabled section.
I think it would be better to leave them as is because the user could ignore them easier if they were not doing a disconnected registry. Placing var in multiple places makes it harder for the user I believe. Thoughts?

Reply:
i just realized i need to change the way the registry_port and registry_dir vars are handled. they need references/set to the hosts[groups[registry_host][0]] method.

Reply:
fixed. in next commit